docs/UserGuide.md

```diff
@@ -167,13 +167,28 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROOM_NUMBER] [a/ADDRESS] [en
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* * Once `ADDRESS`, `ROOM_NUMBER`, `EMERGENCY_NAME`, `EMERGENCY_PHONE`, and/or `GRADUATION_YEAR` has been specified, you cannot remove these fields.
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `t/` without
   specifying any tags after it.
 
+[!Note]
+> * A person can have up to 10 tags (including 0).
+> * `NAME` and `EMERGENCY_NAME` consist of alphabets, numbers, dashes (-) and apostrophes (').
+> * `PHONE` and `EMERGENCY_PHONE_NUMBER` consist of an optional country code indicated with a plus (+), an optional area code and a compulsory number.
+> * `EMAIL` should be of the format `local-part@domain`.
+> * `ROOM_NUMBER` should be of the format `##-####`, where # refers to integer values. 
```

Comment:
Should integer be explained more clearly for non-technical users. Same would apply to local-part and domain above but as that is inherently technical by nature what characters are allowed in local-part and domain should be enough.